dask/dataframe/core.py

```diff
@@ -975,7 +976,7 @@ def rolling(self, window, min_periods=None, freq=None, center=False,
 
         Parameters
         ----------
-        window : int
+        window : int or offset
            Size of the moving window. This is the number of observations used
            for calculating the statistic. The window size must not be so large
            as to span more than one adjacent partition.
```

Comment:
Recommend increasing this to say that it can either be an integer number of rows or a time offset string, like `'10s'`.  As a casual Pandas user I don't know what an offset is necessarily.